namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-staging/resources/rds-prison-visits.tf

```diff
@@ -0,0 +1,43 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "prison-visits-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.2"
+
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "prison-visits-booking"
+  business-unit          = "HMPPS"
+  application            = "prison-visits-booking-staging"
+  is-production          = "false"
+  environment-name       = "staging"
+  infrastructure-support = "pvb-technical-support@digtal.justice.gov.uk"
+  db_engine              = "postgres"
+  db_engine_version      = "9.6"
+  db_name                = "visits"
+  aws_region             = "eu-west-2"
```

Comment:
aws_region is now deprecated. 

Please replace it with a providers block, see the link below for an example: 
[https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance](https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance)